WooCommerce/Classes/ViewRelated/Reviews/ReviewDetailsViewController.swift

```diff
@@ -384,8 +384,13 @@ private extension ReviewDetailsViewController {
             self.moderateReview(siteID: reviewSiteID, reviewID: reviewID, doneStatus: .spam, undoStatus: .unspam)
         }
 
-        commentCell.onReply = {
-            // TODO: Open a text view to send a comment reply to the product review
+        commentCell.onReply = { [weak self] in
+            guard let self = self else {
+                return
+            }
```

Comment:
Should we be using the new `if let` syntax?

Reply:
Sure! I just hadn't updated to Xcode 14 yet when I opened this PR 😅